source/fuzz/transformation_add_loop_to_create_int_constant_synonym.cpp

```diff
@@ -53,8 +53,8 @@ bool TransformationAddLoopToCreateIntConstantSynonym::IsApplicable(
     opt::IRContext* ir_context,
     const TransformationContext& transformation_context) const {
   // Check that |message_.constant_id|, |message_.initial_val_id| and
-  // |message_.step_val_id| are existing constants.
-
+  // |message_.step_val_id| are existing constants, and that their values are
+  // not irrelevant.
```

Comment:
I think for this one, which adds a loop, it's best to only apply it if it can then be useful, so here I think it's right to say that it's inapplicable if the constants are irrelevant.